### ISSUE TITLE
2.0-rc3 is failing to inject jquery

### ISSUE THREAD

**Issue by vegegoku** [GwtMaterialDesign/gwt-material#539]
i am trying to simple test application with GMD but whenever i use version 2.0-rc3 it always throw exception in the browser log

```
ncaught ReferenceError: $ is not defined
    at <anonymous>:1:97
    at fo_g$ (ScriptInjector.java:290)
    at po_g$.qo_g$ [as inject_0_g$] (ScriptInjector.java:88)
    at rZc_g$ (MaterialDesignBase.java:86)
    at qZc_g$ (MaterialDesignBase.java:65)
    at zZc_g$.vZc_g$ [as load_0_g$] (MaterialDesignBase.java:50)
    at zZc_g$.AZc_g$ [as onModuleLoad_0_g$] (MaterialDesign.java:29)
    at Array.cbc_g$ (com_00046_00046security_00046AppClientModule__EntryMethodHolder.java:3)
    at initializeModules_0_g$ (ModuleUtils.java:44)
    at Rp_g$ (Impl.java:239)
    at Up_g$ (Impl.java:298)
    at Impl.java:77
    at vac_g$ (ModuleUtils.java:55)
    at CoreLoggerFactory.java:5
	(anonymous)	@	VM418:1
fo_g$	@	ScriptInjector.java:290
qo_g$	@	ScriptInjector.java:88
rZc_g$	@	MaterialDesignBase.java:86
qZc_g$	@	MaterialDesignBase.java:65
vZc_g$	@	MaterialDesignBase.java:50
AZc_g$	@	MaterialDesign.java:29
cbc_g$	@	com_00046_00046security_00046AppClientModule__EntryMethodHolder.java:3
initializeModules_0_g$	@	ModuleUtils.java:44
Rp_g$	@	Impl.java:239
Up_g$	@	Impl.java:298
(anonymous)	@	Impl.java:77
vac_g$	@	ModuleUtils.java:55
(anonymous)	@	CoreLoggerFactory.java:5
```
when i revert back to 2.0-rc2 everything works fine without errors.

**Comment by BenDol** [comment]
Can you show me your config setup?

**Comment by vegegoku** [comment]
sorry for the delay response

my maven dependencies :

```
        <dependency>
            <groupId>com.github.gwtmaterialdesign</groupId>
            <artifactId>gwt-material</artifactId>
            <version>2.0-rc3</version>
        </dependency>
        <dependency>
            <groupId>com.github.gwtmaterialdesign</groupId>
            <artifactId>gwt-material-addins</artifactId>
            <version>2.0-rc3</version>
        </dependency>
```

in my gwt module xml :

`<inherits name="gwt.material.design.GwtMaterialWithJQuery" />`

thats all.

**Comment by kevzlou7979** [comment]
Can you try the archetype projects https://github.com/GwtMaterialDesign/gwt-material-archetype to check whether its working?

**Comment by persicsb** [comment]
I can confirm this issue. The archetype does not even create a valid project.

**Comment by BenDol** [comment]
Would one of your guys be able to set up a small project that we can reproduce this issue in? I am unsuccessful in producing the error you are having. Thanks!

**Comment by BenDol** [comment]
I have added some potential fixes, are you able to test the latest 2.0-SNAPSHOT?

**Comment by kevzlou7979** [comment]
Any follow up test on 2.0-SNAPSHOT @persicsb. As on my side I haven't reproduce this issue.
